### PR TITLE
add support for camera and location section

### PR DESCRIPTION
**Pull request by CalebMuhia**

### CODE REVIEW

File: source/index.html.md [hunk 1]
```diff
@@ -216,6 +216,39 @@ score | Score of the plate reader text. Range [0, 1].
...
+  -F regions=fr -F regions=us-ca \
+  -F camera_id=123 \
+  -F latitude=1.23366 \
+  -F longitude=-12.44556 \
+  -H 'Authorization: Token API_TOKEN' \
+  https://api.platerecognizer.com/v1/plate-reader-dashboard/
+
+```
+
+This endpoint suports additional camera and location parameters.
```
Comment (marcbelmont) [inline source/index.html.md]: su**p**ports

File: source/index.html.md [hunk 2]
```diff
@@ -216,6 +216,39 @@ score | Score of the plate reader text. Range [0, 1].
...
+Parameter | Required | Description
+--------- | ----------- | -----------
+upload | Yes |The file to be uploaded
+regions | No | Match the license plate pattern of specific [regions](#regions-supported). This parameter can be used 
+camera_id | Yes | unique ID for camera
+latitude | Yes | Latitude location of the camera
+longitude | Yes | Logitude location of the camera
+
+<aside class="notice">
+Support for camera and location is currently in beta testing and is only available on invite-only basis.
```
Comment (marcbelmont) [inline source/index.html.md:248]: Add:
[Contact us](https://platerecognizer.com/contact/) is you are interested in using this feature.

File: source/index.html.md [hunk 3]
```diff
@@ -216,6 +216,39 @@ score | Score of the plate reader text. Range [0, 1].
...
+`POST https://api.platerecognizer.com/v1/plate-reader-dashboard/`
+
+### POST Parameters
+
+Parameter | Required | Description
+--------- | ----------- | -----------
+upload | Yes |The file to be uploaded
+regions | No | Match the license plate pattern of specific [regions](#regions-supported). This parameter can be used 
+camera_id | Yes | unique ID for camera
+latitude | Yes | Latitude location of the camera
```
Comment (marcbelmont) [inline source/index.html.md]: latitude and longitude are not required

File: source/index.html.md (hunk 1 above)
Comment (CalebMuhia) [reply source/index.html.md]: fixed

File: source/index.html.md (hunk 2 above)
Comment (CalebMuhia) [reply source/index.html.md:248]: fixed

File: source/index.html.md (hunk 3 above)
Comment (CalebMuhia) [reply source/index.html.md]: fixed

**Pull request merged**